apps/server/src/modules/files-storage/uc/file-record.uc.ts

```diff
@@ -18,31 +18,50 @@ export class FileRecordUC {
 		private readonly authorizationService: AuthorizationService
 	) {}
 
-	async patchFilename(userId: EntityId, params: SingleFileParams, data: RenameFileParams) {
+	private checkDouplicatedNames(fileRecords: FileRecord[], data: RenameFileParams): void | ConflictException {
```

Comment:
There's a typo here. It should be "checkDuplicatedNames".

Reply:
fixed